.genie/wishes/genie-simulations/WISH.md

```
@@ -0,0 +1,488 @@
+
+## Execution Groups
+
+### Group 1: PG Schema — Simulation Tables
+**Goal:** Create the database schema for storing simulation runs, scenarios, turns, scores, and annotations.
+
+**Deliverables:**
+1. Migration file creating 5 tables: `genie_sim_runs`, `genie_sim_scenarios`, `genie_sim_turns`, `genie_sim_scores`, `genie_sim_annotations`
+2. Indexes for common queries: runs by agent, scenarios by run, turns by scenario, annotations by turn
+
+**Schema:**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)